ci/ray_ci/ray_docker_container.py

```diff
@@ -0,0 +1,58 @@
+import os
+from typing import List
+
+from ci.ray_ci.container import _DOCKER_ECR_REPO
+from ci.ray_ci.docker_container import DockerContainer
+from ci.ray_ci.builder_container import PYTHON_VERSIONS
+from ci.ray_ci.utils import docker_pull, RAY_VERSION, POSTMERGE_PIPELINE
+
+
+class RayDockerContainer(DockerContainer):
+    """
+    Container for building and publishing ray docker images
+    """
+
+    def run(self) -> None:
```

Comment:
none of these are new; they are moved from docker_container to create a subclass